Honda: Update Clarity specs

roenthomas opened the pull request:
Unsure why these values were changed when ported over.

## Summary by Sourcery

Bug Fixes:
- Restore Honda Clarity mass to 1838 kg, revert minimum steer speed to 3 mph, and specify 'All' trim in documentation

roenthomas committed:
Update Clarity specs in values.py to match prior values

Unsure why these values were changed when ported over.

roenthomas committed:
Merge branch 'master' into master-PR7

sunnyhaibin commented on opendbc/car/honda/values.py:
```diff
@@ -351,8 +351,8 @@ class CAR(Platforms):
-    [HondaCarDocs("Honda Clarity 2018-21", min_steer_speed=12. * CV.MPH_TO_MS)],
-    CarSpecs(mass=1834, wheelbase=2.75, centerToFrontRatio=0.4, steerRatio=16.5),
+    [HondaCarDocs("Honda Clarity 2018-21", min_steer_speed=3. * CV.MPH_TO_MS)],
+    CarSpecs(mass=4052. * CV.LB_TO_KG, wheelbase=2.75, centerToFrontRatio=0.41, steerRatio=16.50, tireStiffnessFactor=1.),
```
Should probably align with this instead: https://github.com/sunnypilot/sunnypilot/blob/b2cdfc09aec013ea81841e30f9b9a00b3fbf6fc1/selfdrive/car/honda/values.py#L278

roenthomas replied:
Feedback implemented

sunnyhaibin reviewed (changes requested):
Converting to Draft until the items are addressed.

roenthomas committed:
Merge branch 'sunnypilot:master' into master-PR7

roenthomas committed:
Corrected import to align with vanillagorillaa clarity_0.10.0

VG is the originator of all clarity ports

roenthomas committed:
Merge branch 'master-PR7' of https://github.com/roenthomas/Roendbc into master-PR7

sunnyhaibin commented on opendbc/car/honda/values.py:
```diff
@@ -351,8 +351,8 @@ class CAR(Platforms):
-    [HondaCarDocs("Honda Clarity 2018-21", min_steer_speed=12. * CV.MPH_TO_MS)],
-    CarSpecs(mass=1834, wheelbase=2.75, centerToFrontRatio=0.4, steerRatio=16.5),
+    [HondaCarDocs("Honda Clarity 2018-2021", "All", min_steer_speed=3. * CV.MPH_TO_MS)],
+    CarSpecs(mass=4052. * CV.LB_TO_KG, wheelbase=2.75, centerToFrontRatio=0.4, steerRatio=16.50),
```
```suggestion
    CarSpecs(mass=1838, wheelbase=2.75, centerToFrontRatio=0.4, steerRatio=16.5),
```
 
- `mass`: 1838 kg = 4052 lb. No functional benefit, using KG in mass is preferred.
- `centerToFrontRatio`: a 0.01 difference has negligible performance impact, but we can merge this if this in practice significant changes performance.
- `steerRatio`: 16.5 = 16.50. No functional benefit.

roenthomas replied:
Feedback implemented

sunnyhaibin committed:
Merge branch 'master' into master-PR7

roenthomas committed:
Merge branch 'master' into master-PR7

roenthomas committed:
Implemented feedback from PR discussion

Reduced to 2 digit trailing year, mass in kg, min_steer_speed=3 per uploaded route.

THERoen commented on opendbc/car/honda/values.py:
```diff
@@ -351,8 +351,8 @@ class CAR(Platforms):
   HONDA_CLARITY = HondaNidecPlatformConfig(
-    [HondaCarDocs("Honda Clarity 2018-21", min_steer_speed=12. * CV.MPH_TO_MS)],
-    CarSpecs(mass=1834, wheelbase=2.75, centerToFrontRatio=0.4, steerRatio=16.5),
+    [HondaCarDocs("Honda Clarity 2018-2021", "All", min_steer_speed=3. * CV.MPH_TO_MS)],
```
Feedback implemented in latest commit

roenthomas committed:
Merge branch 'master' into master-PR7

sunnyhaibin commented:
> Was the min steer speed change PR'ed?

Which PR?

sunnyhaibin committed:
Merge branch 'master' into master-PR7

roenthomas commented:
> > Was the min steer speed change PR'ed?
> 
> Which PR?

This one, changing the min steer values for stock Clarity down to 3 mph.

What's preventing this PR from being merged?

sunnyhaibin committed:
Merge branch 'master' into master-PR7

sunnyhaibin committed:
fix ci

sunnyhaibin committed:
Merge branch 'master' into master-PR7

sunnyhaibin committed:
bruh
